### PR TITLE
Scaffold out pypy publish

### LINKED ISSUE #7: Leverage Github for releases
It would be neat to leverage Github releases when releasing new code to be notified when new releases happen.

Could leverage Github Actions to automatically push to pipy too.

Happy to help out on this one.

### PR DESCRIPTION
**Pull request by inverse**
Resolves #7 

- Triggers on tag ( need to test the rule) documentation is here: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags

~~The only part that would be great to figure out is how to trigger the normal ci pipeline so that it runs the full linting/testing before publish.~~ This isn't a blocker since the CI pipeline will have kicked in for any push/merge to master but will assume that it's run and succeeded.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Package
+      run: |
+        python setup.py sdist bdist_wheel
+    - name: Publish to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.pypi_password }}
```
Comment (inverse) [inline .github/workflows/publish.yml:28]: Will require you setting this in the repo settings under secrets.

Further information around generating this: https://github.com/pypa/gh-action-pypi-publish#usage

Comment (valayDave) [reply .github/workflows/publish.yml:28]: Will Get it done over the next few days.Thanks.

Comment (inverse) [reply .github/workflows/publish.yml:28]: No rush - but let me know if there is anything that can be done to help.

**Comment by inverse** [comment]
@valayDave I don' think you need to change the `user`.

see: https://pypi.org/help/#apitoken

**Comment by valayDave** [comment]
I figured it. I miss read docs.  Added correct token and setup secrets.

**Comment by valayDave** [comment]
@inverse FIxed token issue and added secret. On approval, I will merge and push tag

**Comment by inverse** [comment]
@valayDave what do you mean on approval? Something I need to do? Changes look reasonable.

**Pull request merged**